Npm Dependency Version Service Test Failing

:clock11: **When did the problem start?**
~2 days ago (as of Feb 27th)

<!-- Indicate when the problem started -->

:camera: **Live badge**
https://img.shields.io/npm/dependency-version/mocha/dev/@mocha/docdash
![](https://img.shields.io/npm/dependency-version/mocha/dev/@mocha/docdash)

:wrench: **Is the live badge working?**
Yup

<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://circleci.com/gh/badges/daily-tests/944
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
NpmDependencyVersion [live] gets the dev dependency version (scoped dependency) [ GET /mocha/dev/@mocha/contributors.json ]
  AssertionError: label mismatch: expected 'dependency' to equal '@mocha/contributors'
```

:bulb: **Possible solution**
Looks like v7.1.0 of mocha (which was published 2 days ago) has includes the removal of `@mocha/contributors` from the dev dependency list which is why the test is failing. Will open a PR with a new test target
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/npm/npm-dependency-version.tester.js
+++ b/services/npm/npm-dependency-version.tester.js
@@ -25,9 +25,9 @@ t.create('gets the dev dependency version (scoped)')
   })
 
 t.create('gets the dev dependency version (scoped dependency)')
-  .get('/mocha/dev/@mocha/contributors.json')
+  .get('/mocha/dev/@mocha/docdash.json')
   .expectBadge({
-    label: '@mocha/contributors',
+    label: '@mocha/docdash',
     message: semverRange,
   })
 

